docs/pages/inboxes/push-notifications.md

```diff
@@ -0,0 +1,99 @@
+```
+
+*Replaces V2 `/xmtp/0/m-$addresses/proto`*
+
+### Subscribe to topics
+
+Subscribe to all relevant topics, allowing your app to monitor for push notifications about both new and ongoing conversations. 
+
+This code sample retrieves all topics associated with `alix`’s conversations, for example, enabling the app to receive push notifications only for conversations in which `alix` is a part of.
+
+```tsx [React Native]
+const conversations = await alix.conversations.listConversations()
+const topics = conversations.map((conv: any) => conv.topic)
+
+await subscribeAll([alix.welcomeTopic(), ...topics])
+```
+
+### Receive push notifications
+
+On receipt of a push notification, decode it:
+
+```tsx [React Native]
+const receivedBytes = Buffer.from(received.message, 'base64').toString('utf-8')
+```
+
+Then determine whether it’s for a new conversation or an existing one.
+
+- **If it’s a Welcome message** (`alix.welcomeTopic() == received.topic`), initiate the conversation with `conversationFromWelcome`:
+    
+    ```tsx [React Native]
+    const conversation = await alix.conversations.conversationFromWelcome(receivedBytes)
+    ```
+    
+- **If it’s a V3 message**, find the corresponding topic, sync the conversation, and process the new message:
+    
+    ```tsx [React Native]
+    const conversation = await alix.findConversationByTopic(received.topic)
+    await conversation.sync()
+    const message = await conversation.processMessage(receivedBytes)
+    ```
+    
+
+### Run a push notification server
+
+To implement push notifications in your app, you must run a push notification server. This server acts as an intermediary between the XMTP network and your app’s push notification service, ensuring that users receive timely and relevant notifications.
+
+**Why is a push notification server required?**
+
+- **Continuous monitoring**: The XMTP network operates on a decentralized protocol, where messages are exchanged directly between clients. However, for push notifications, your app needs a dedicated server to continuously listen for new messages or events on the XMTP network, even when the user’s device is offline or the app is not running.
+- **Message processing**: Upon detecting new messages or events, the server processes them to determine their relevance and formats them appropriately for push notifications. This includes extracting necessary information, such as the sender’s identity and message content, to craft meaningful notifications.
+- **Integration with push notification services**: The server interfaces with platform-specific push notification services, like [Apple Push Notification Service](/notifications/build-notifications#understand-apple-entitlements-for-ios-apps) (APNs) for iOS or Firebase Cloud Messaging (FCM) for Android. It sends the processed notifications to these services, which then deliver them to the user’s device.
+
+To learn more about running a push notification server, see:
+
```

Comment:
Valid resources? Or  maybe we shouldn't point to the Android and iOS examples if they haven't been updated to v3 yet?